package.json

```diff
@@ -1,102 +1,30 @@
 {
   "name": "faunadb",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
Should we increase the package version now or let we do a separated PR once we decide to publish it?